projects/bifold-wallet/README.md

```diff
@@ -0,0 +1,133 @@
+# Project Name 
+
+Right now, we often refer to our project as "Aries Bifold," but its official name on GitHub is [Aries Mobile Agent React Native](https://github.com/hyperledger/aries-mobile-agent-react-native). If the OpenWallet Foundation (OWF) accepts our project, we're likely change its official name to either "Aries Bifold" or possibly substitute the "Aries" part altogether. This would help us differentiate our project from the "Aries Hyperledger" initiative. We plan to start discussing this name change as soon as we are part of the OWF. 
```

Comment:
Since Aries is part of the Hyperledger world, I would suggest dropping Aries from the name. Whatever name is decided, we should make sure to run this past LF Legal.

Reply:
I think we can happily drop "Aries" and stick with just "Bifold". Bifold is a pretty generic "style" of wallet so I would think it immune from trademarking. It would be like trademarking the word "hamburger".